tests/scanner/scanners/external_project_access_scanner_test.py

```diff
@@ -239,51 +240,9 @@ def test_get_emails(self, mock_get_latest_inv_ndx_id, mock_storage):
         self.assertListEqual(emails, expected_emails)
 
 
-class OtherModuleTest(ForsetiTestCase):
-    """Test misc."""
-
-    def setUp(self):
-        """Set up."""
-        fake_global_configs = {
-            'crm': {'max_calls': 4, 'period': 1.2}}
-
-        mock.patch.object(google.auth, 'default', return_value=(
-            mock.Mock(spec_set=credentials.Credentials), 'test-project'))
-
-        mock_creds = api_helpers.get_delegated_credential('user1@eaxmple.com')
-
-        self.crm_api_client = crm.CloudResourceManagerClient(
-            global_configs=fake_global_configs,
-            use_rate_limiter=False,
-            credentials=mock_creds)
-        self.project_id = fake_crm_responses.FAKE_PROJECT_ID
-
-    def test_get_project_ids(self):
-        """Test get_projects_ids() for all lifecycle_states."""
-        response = json.dumps(fake_crm_responses.FAKE_PROJECTS_API_RESPONSE1)
-        http_mocks.mock_http_response(response)
-
-        proj_ids = epas.extract_project_ids(self.crm_api_client)
-
-        self.assertListEqual(
-            proj_ids,
-            fake_crm_responses.EXPECTED_FAKE_PROJECTS_API_RESPONSE1_IDS)
-
-    def test_cast_to_gcp_resources(self):
-        """Validate get_project_ancestry_resources() with test project."""
-        http_mocks.mock_http_response(
-            fake_crm_responses.GET_PROJECT_ANCESTRY_RESPONSE)
-        result = self.crm_api_client.get_project_ancestry(
-            fake_crm_responses.FAKE_PROJECT_ID)
-
-        cast_resources = resource_util.cast_to_gcp_resources(result)
-
-        self.assertTrue(isinstance(cast_resources, list))
 
-        self.assertTrue(isinstance(cast_resources[0], Project))
-        self.assertTrue(isinstance(cast_resources[-1], Organization))
 
-def reduced_inventory(storage, types):
+def get_inventory(storage, types):
```

Comment:
Is this `reduced_inventory()` being called from somewhere?  Do we need to change the caller?

Reply:
deleted